### ISSUE TITLE
Create "current situation in Belgium" component

### ISSUE THREAD

**Issue by niconoe** [inbo/crow#91]
It should show:

- Main direction
- Number of birds in last night
- ~Number of birds in the last hour~
- ~Maybe map with 4 radars~

Data source: average of density and heading from 4 Belgian radars (last day)

Possibly, imagine how that can be implemented server-side (preprocessing) if the number of radar grows.

**Comment by peterdesmet** [comment]
In the current application I have boiled this down to one number, in the first sentence of the description.

> x birds were detected by weather radars over Belgium last night.

I'm fine with keeping it this limited. We could potentially extend it with a main direction, but I think adding a full extra component would make it more cluttered and take away from the main visualization. What remains:

- Calculation duration of last night: beginning dusk -> end dawn?
- Calculate number of birds and sum them for the 4 Belgian radars
- Calculate weighted direction?
- Test how fast this can be done

**Comment by niconoe** [comment]
I'd like to start implementing this, but wonder about the extra load (4 radars * 2 files for each CROW visitor) on KMI's servers.

I remember @mrtnrey mentioning on another issue that it was important to stay as light as possible. 

We could possibly implement a tiny web service that grab the radar files and perform the calculation only once, then serve its result to all CROW visitors. Better for the network load, but we managed so far to develop CROW without any server-side component, which is nicer in terms of long term maintenance.

So, I'm not sure about this. Any opinion, @mrtnrey and @peterdesmet?

**Comment by mrtnrey** [comment]
Hmm, difficult issue.
Yes we should keep the traffic light, not only for the KMI servers but also for the app to load. Calculating the "gross number" of birds travelling through BE might jeopardize this intention.
But remember that the app will primarily be installed at meteo.be itself (probably meteo.be/birddetection ) so the traffic will be local, no?
Another issue arises if the app will be used in the next phase (after CROW) as the standard visualisation tool for all EU radar in the GloBAM project? I thought it was Peter's intention to do so. We should pay attention that the app is not too BE-specific, so that it can be easily reused in another context / country.
Is it okay to discuss this further on our meeting of next week?

**Comment by niconoe** [comment]
About the traffic, the requests will originate from the visitor's browsers, even if the HTML/JS is hosted at meteo.be.

My general impression about extending this to GloBAM is that there will be indeed some work involved (at least to decouple from Belgium, and to deal with more data => maybe server-side components will be finally needed), but at least we've build a lot of experience and already have some decently running code.

Good idea to discuss that in next week's meeting!

**Comment by niconoe** [comment]
Meeting decision: let's add 1 sentence: "x birds detected during the selected period by the selected radar"
(we could also quite easily separate the day/night values if we want)

(no more data load necessary)

**Comment by niconoe** [comment]
Blocked by #112

**Comment by peterdesmet** [comment]
The calculation and phrasing is discussed in #112, so this issue has become a duplicate.
